cypress/e2e/buy-shirt.cy.ts

```diff
@@ -17,6 +17,7 @@ const paymentStep = new PaymentStep();
 
 describe("Buy a t-shirt", () => {
   it("then the t-shirt should be bought", () => {
```

Comment:
you can add a before hook and initialize page objects in a similar way that is done in cypress/e2e/dresses-list.cy.ts file